package.json

```diff
@@ -2,9 +2,12 @@
   "name": "stylis-plugin-rtl",
   "version": "1.0.0",
   "description": "Fork of stylis-rtl, uses cssjanus under the hood to flip style orientations for RTL",
-  "main": "dist/stylis-rtl.js",
+  "main": "dist/lib/stylis-rtl.js",
```

Comment:
let's keep the directory flat and just do `stylis-rtl.cjs.js` and `stylis-rtl.esm.js`

Reply:
![image](https://user-images.githubusercontent.com/14183168/91539631-e72a8880-e919-11ea-9dab-132167bf33fc.png)

Done 👍